src/redux/crypto/cryptoSlice.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import { selectSearchTerm } from '../search/searchSlice';
+
+axios.defaults.baseURL = 'https://api.coincap.io/v2/assets';
+
+/* axios.defaults.headers.common['Authorization'] = process.env.TOKEN; */
+/* const baseURL = 'https://api.coincap.io/v2/assets'; */
```

Comment:
- [ ] OPTIONAL
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing your code to the repository so as to keep your code well organized.